GDM.HW4.OOP.Classes.Task1/Program.cs

```diff
@@ -0,0 +1,143 @@
+            Console.WriteLine(pleyer + " 1:\n" + Sony.GetDescription());
+            Console.WriteLine(pleyer + " 2:\n" + Panasonic.GetDescription());
+
+
+            Console.ReadLine();
+        }
+    }
+    class LapTop
+    {
+        public string Name;
+        public string Power;
+        public string Memory;
+        public string Waight;
+
+        public LapTop(string name, int power, int memory, double waight)
+        {
+            Name = name;
+            Power = $"{power} W";
+            Memory = $"{memory} Gb";
+            Waight = $"{waight} Kg";
+        }
+        public string GetDescription()
+        {
+            return $"  Name: {Name} {Environment.NewLine}  Power: {Power} {Environment.NewLine}  Memory: {Memory} {Environment.NewLine}  Waight: {Waight}";
+        }
+    }
+    class Server
+    {
+        public string Name;
+        public string Power;
+        public string Memory;
+        public int ProcessorQuantity;
+
+        public Server(string name, int power, int memory, int processorQuantity)
+        {
+            Name = name;
+            Power = $"{power} W";
+            Memory = $"{memory} Gb";
+            ProcessorQuantity = processorQuantity;
+        }
+        public string GetDescription()
+        {
+            return $"  Name: {Name} {Environment.NewLine}  Power: {Power} {Environment.NewLine}  Memory: {Memory} {Environment.NewLine}  Processors Quantity: {ProcessorQuantity}";
+        }
+    }
+    class PlasmTV
+    {
+        public string Name;
+        public string Power;
+        public string ScreenDiagonal;
+        public string ScreenResolution;
+
+        public PlasmTV(string name, int power, int screenDiagonal, string screenResolution)
+        {
+            Name = name;
+            Power = $"{power} W";
+            ScreenDiagonal = $"{screenDiagonal}''";
+            ScreenResolution = $"{screenResolution} Px";
+        }
+        public string GetDescription()
+        {
+            return $"  Name: {Name} {Environment.NewLine}  Power: {Power} {Environment.NewLine}  Screen Diagonal: {ScreenDiagonal} {Environment.NewLine}  Screen Resolution: {ScreenResolution}";
+        }
+    }
+    class LEDTV
+    {
+        public string Name;
+        public string Power;
+        public string ScreenDiagonal;
+        public string Frequency;
+
+        public LEDTV(string name, int power, int screenDiagonal, int frequency)
+        {
+            Name = name;
+            Power = $"{power} W";
+            ScreenDiagonal = $"{screenDiagonal}''";
+            Frequency = $"{frequency} Hz";
+        }
+        public string GetDescription()
+        {
+            return $"  Name: {Name} {Environment.NewLine}  Power: {Power} {Environment.NewLine}  ScreenDiagonal: {ScreenDiagonal} {Environment.NewLine}  Frequency: {Frequency}";
+        }
+    }
+    class PleyerDevice
+    {
+        public string Name;
+        public string Power;
+        public string MP3;
```

Comment:
Bad solution. You hardcoded types. You should create one field/property for types.